File: w3d_translator/classes/object.py
class Object:
    def __init__(
        self,
        name: str,
        visible: bool,
        color,
        lighting: bool,
        clickthrough: bool,
        around_self_axis: bool,
        scale: float,
        placement,
        content,
        links: list,
        text: str,
        sound_ref: str,
    ):
        self.name = name
        self.visible = visible
        self.color = color  # TODO: Custom Unity color type
        self.lighting = lighting
        self.clickthrough = clickthrough
        self.around_self_axis = around_self_axis
        self.scale = scale
        self.placement = placement  # TODO: Placement object type
        self.content = content  # TODO: Content object type
        self.links = links  # TODO: Array of link type
        self.text = text
        self.sound_ref = sound_ref

File: classes/Object.cs
[XmlRoot(ElementName="Object")]
public class Object { 

	[XmlElement(ElementName="Visible")] 
	public bool Visible; 

	[XmlElement(ElementName="Color")] 
	public double Color; 

	[XmlElement(ElementName="Lighting")] 
	public bool Lighting; 

	[XmlElement(ElementName="ClickThrough")] 
	public bool ClickThrough; 

	[XmlElement(ElementName="AroundSelfAxis")] 
	public bool AroundSelfAxis; 

	[XmlElement(ElementName="Scale")] 
	public double Scale; 

	[XmlElement(ElementName="Placement")] 
	public Placement Placement; 

	[XmlElement(ElementName="Content")] 
	public Content Content; 

	[XmlAttribute(AttributeName="name")] 
	public string Name; 

	[XmlText] 
	public string Text; 

	[XmlElement(ElementName="LinkRoot")] 
	public LinkRoot LinkRoot; 

	[XmlElement(ElementName="SoundRef")] 
	public string SoundRef; 
}
